File: pytools/data/data.code.csharp.client/AckSceneEnter.cs
using System.Collections;
using System.Collections.Generic;


public class AckSceneEnter
{
	private MsgScenePlayer _player;


	public AckSceneEnter(Packet packet)
	{
		this._player = new MsgScenePlayer(packet);
	}


	public MsgScenePlayer player
	{
		get { return this._player; }
		set { this._player = value; }
	}

}

File: pytools/examples/golang/src/client/proto/ack_scene_enter.go
package proto

import (
	"packet"
)

type AckSceneEnter struct {
	player                   *MsgScenePlayer
}

func AckSceneEnterDecode(pack *packet.Packet) *AckSceneEnter {
	ackSceneEnter := &AckSceneEnter{}

	ackSceneEnter.player = MsgScenePlayerDecode(pack)
	return ackSceneEnter
}

func (this *AckSceneEnter) GetPlayer() *MsgScenePlayer {
	return this.player
}
